src/smc-webapp/smc-react.coffee

```diff
@@ -592,7 +592,7 @@ exports.ReactDOM = require('react-dom')
 if DEBUG
     smc?.redux = redux  # for convenience in the browser (mainly for debugging)
 
-_internals =
+exports._internals =
```

Comment:
I understand this is only used in https://github.com/sagemathinc/cocalc/blob/9fa215ca64f6ec824e807ac21fbef8bf5bb509c5/src/smc-webapp/test/smc-react-test.coffee#L2 

and well, otherwise it won't work. 

I don't think there are any negative implications exporting it. Still, in case there is paranoia about this, we could conditionally do it via `if process?.env.SMC_TEST == 'true'` …

Reply:
done